core/src/com/unciv/models/ruleset/unique/UniqueTriggerActivation.kt

```diff
@@ -41,16 +41,20 @@ object UniqueTriggerActivation {
         notification: String? = null,
         triggerNotificationText: String? = null
     ): Boolean {
+
+        val relevantCity by lazy {
+            city?: tile?.getCity()
```

Comment:
Since this is definitely in use at line 55, we can put it at 54 and have it not be lazy

Reply:
Not sure where it being lazy is relevant

Actually, huh. On line 55 it's redundant. `conditionslsApply` has the same relevant city check. We still need to do it twice so where it's placed I doubt matters

~~Also, why are we checking conditions applies here? I know I just PR'es to check this at the source of the trigger rather than in here... Which apparently was redundant for non unit here... But the check here still feels off. Must just be me, Idk~~